crates/services/gas_price_service/src/fuel_gas_price_updater.rs

```diff
@@ -104,6 +113,18 @@ pub trait L2BlockSource: Send + Sync {
     async fn get_l2_block(&mut self, height: BlockHeight) -> Result<BlockInfo>;
 }
 
+#[derive(Debug, Default)]
+pub struct DaCommitDetails {
+    pub l2_block_range: core::ops::Range<u32>,
+    pub blob_bytes: u32,
```

Comment:
Shouldn't the blob bytes be a `Vec<u8>`? Or is this the number of bytes? Then i'd suggest renaming it to `blob_size`.

Reply:
Given that we're probably not interested in the blob bytes here, I assume from the context that this is the blob size. It would be good to avoid any confusion to have the name reflect that.

Reply:
addressed in 350c7ccf8